R/ncdf.R

```diff
@@ -631,14 +659,19 @@ read_ncdf = function(.x, ..., var = NULL, ncsub = NULL, curvilinear = character(
   ## cannot assume we have COORDS style coordinate varibales
   ## - so create them as 1:length if needed
   coords = setNames(vector("list", length(dims$name)), dims$name)
+  
+  attr(coords, "cv") <- c()
```

Comment:
A refactor would make this attribute unnecessary, but I just use the attribute to pass the coordinate variable name that actually got used back up to the calling environment.